fluent/client/client.go

```diff
@@ -47,7 +53,6 @@ type Session struct {
 //counterfeiter:generate . ConnectionFactory
 type ConnectionFactory interface {
 	New() (net.Conn, error)
-	Session() (*Session, error)
```

Comment:
Removed because it is unused and doesn't match what was needed for WSConnectionFactory